### PR TITLE
Removal of SVN ID line as it is not needed.

### PR DESCRIPTION
**Pull request by sdcloudt**
SVN is not used anymore. Therefore the `/* $Id$ */` lines are not necessary. This PR removes most, if not all, id lines.

I checked the commit, but I would appreciate as checking 1203 files is a job which I nobody wants to do.

I used the following line of bash code to remove all `/* $Id$ */` lines:
```
for file in **/*
do
  sed -Ei '1{N; /^\/\*\s\$Id\$\s\*\/\n$/d}' $file
done
```

### CODE REVIEW

**Comment by glx22** [comment]
*.sq files are generated, you'll need to change https://github.com/OpenTTD/OpenTTD/blob/f1204f0d9dafc0bcdfaa2872ed48835afce13ddb/src/script/api/squirrel_export.awk#L57-L58 and https://github.com/OpenTTD/OpenTTD/blob/f1204f0d9dafc0bcdfaa2872ed48835afce13ddb/src/script/api/squirrel_export.vbs#L86-L87

**Comment by LordAro** [comment]
Also remove that merge commit, not sure how it got past the checker...

**Comment by glx22** [comment]
The checker accepts it because we rebase before the check so it just disappear I think

**Comment by sdcloudt** [comment]
I screwed up the git, I am fixing it.

**Comment by sdcloudt** [comment]
I removed some more $Id$ tags, and, removed the generated $Id$ output.

File: media/extra_grf/shore.nfo [hunk 1]
```diff
@@ -1,6 +1,4 @@
 //
```
Comment (LordAro) [inline media/extra_grf/shore.nfo]: This line can probably be removed. Seems common to all the nfo files

Comment (sdcloudt) [reply media/extra_grf/shore.nfo]: Done

**Review by LordAro — APPROVED** [review]

**Pull request merged**